ddtrace/contrib/internal/litellm/utils.py

```diff
@@ -41,7 +41,7 @@ def _finish_spans(self):
                 formatted_completions = _process_finished_stream(
                     self._dd_integration, span, kwargs, self._streamed_chunks, span.resource
                 )
-            elif self._dd_integration.is_pc_sampled_llmobs(span):
+            else:
```

Comment:
is this else necessary? Can we just remove the else clause and execute llmobs_set_tags() always? cc @ncybul